RestApi/Sepes.RestApi/Controllers/StudyController.cs

```diff
@@ -1,70 +1,51 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Cors;
-using Newtonsoft.Json.Linq;
 using Sepes.RestApi.Model;
 using Sepes.RestApi.Services;
+using System;
+using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
 
 namespace Sepes.RestApi.Controller
 {
     [Route("api/[controller]")]
     [ApiController]
     [EnableCors("_myAllowSpecificOrigins")]
-    //[Authorize]
+    [Authorize]
     public class StudyController : ControllerBase
     {
-        private ISepesDb sepesDb;
+        private ISepesDb _sepesDb;
 
-        public StudyController(ISepesDb dbService) {
-            sepesDb = dbService;
+        public StudyController(ISepesDb dbService)
+        {
+            _sepesDb = dbService;
         }
-        
+
         //Create study
         [HttpPost("create")]
-        public int CreationVars([FromBody] Study value)
+        public async Task<ActionResult<int>> CreationVars([FromBody] Study value)
         {
-            return sepesDb.createStudy(value);
+            return Ok(await _sepesDb.createStudy(value.studyName, value.userIds, value.datasetIds));
```

Comment:
Do not need to be ActionResult or use Ok(). As it have no special error handling.